src/components/Chart/getOutlierSeries.test.tsx

```diff
@@ -0,0 +1,89 @@
+import { splitOutlierSeries } from './getOutlierSeries'
+import { AxisDomain, ChartSeries } from './model'
+
+describe('splitOutlierSeries', () => {
+  const yDomain: AxisDomain = { init: 0, end: 100, step: 20 }
+  const xDomain: AxisDomain = { init: 0, end: 5, step: 1 }
+  const domainPoints: number[] = [0, 1, 2, 3, 4, 5]
+  const chartSeries: ChartSeries<number>[] = [{ name: 'test', data: [20, 50, 120] }]
+
+  it.each(['auto', 'expand-domain'] as const)(
```

Comment:
Why this `as const` is necessary?

Reply:
I change for use the `OutliersType`, so its not more necessary